<?xml version="1.0"?>
<launch>
    <arg name="fcu" default="/dev/ttyACM0:57600" />
    <include file="$(find mavros)/launch/node.launch">
        <arg name="pluginlists_yaml" value="$(find foresight)/param/px4_plugins.yaml"/>
        <arg name="config_yaml" value="$(find foresight)/param/px4_config.yaml"/>
        <arg name="fcu_url" value="$(arg fcu)" />
        <arg name="gcs_url" value="udp://:14556@localhost:14550" />
        <arg name="tgt_system" value="1" />
        <arg name="tgt_component" value="1" />
        <arg name="log_output" value="screen" />
    </include>
<!--     <include file="$(find foresight)/launch/localization/static_transformations.launch"/> -->
<!--     <include file="$(find foresight)/launch/localization/ukf_fuser.launch" /> -->

    <node name="imu_tf" pkg="foresight" type="imu_tf.py" args="--test"/>
</launch>
